packages/shared/src/components/modals/ShareModal.tsx

```diff
@@ -53,11 +61,6 @@ export default function ShareModal({
       }),
     );
 
-  const trackAndCopyLink = () => {
-    baseTrackingEvent('share post', { provider: ShareProvider.CopyLink });
-    copyUrl();
-  };
-
```

Comment:
Was it intended to remove the tracking here? @TomDowling @rebelchris

Reply:
We should not remove any tracking for this PR as far as I'm concerned.

Reply:
Thought so too. Let me bring this back.

Reply:
This was removed as the copy link is now within the SocialShare component instead of direct on the ShareModal

Reply:
Ah, I see. Thought it was totally removed. Thanks for the note!